test/engine/func_index.result

```diff
@@ -292,7 +292,8 @@ e = e.prev
  | ...
 e:unpack()
  | ---
- | - base_type: LuajitError
+ | - code: 0
```

Comment:
This would break people who created errors using `box.error.new()` - they have code 0 unless you specified other value, and it is always visible as `box.error.UNKNOWN`. I thought about using -1, but it seems our API allows values `< 0` as a code as well. So I decided to give up and let the code always exist. Let me know if you still want to hide it somehow and I will try to use `-1` as code absence.

Reply:
Okay, let's keep it simple then. I think showing 0 error code (which translates to UNKNOWN) for LuajitError should be fine. No need to complicate things to hide it.